internal/pkg/cli/storage_init.go

```diff
@@ -686,6 +687,7 @@ func (o *initStorageOpts) addonBlobs() ([]addonBlob, error) {
 			name:        o.storageName,
```

Comment:
nit: why not have the name be `fmt.Sprintf("%s.yml", o.storageName)`?

Reply:
The intention was that perhaps we will support non-CFN addons in the future. Then the addon itself needs to decide what extension it's supposed to be. But yea, this is probably a premature generalization. Changed it to `".yml"`!